src/prettify.js

```diff
@@ -1270,6 +1270,15 @@ var prettyPrint;
     }
     return langHandlerRegistry[extension];
   }
+  // It would be silly to add prettyprint to something marked lang-none, but if
+  // some code does so carelessly, it's also silly to highlight it.
+  registerLangHandler(
+      createSimpleLexer(
+          [],
+          [
+           [PR_PLAIN, /^.+/],
```

Comment:
I think I do want plain instead of nocode. Plain seems to be a clearer indication of "no highlighting". In the tests, at least, nocode is distinguished with a grey backround to e.g. set line numbers apart from the code.

Using `[\s\S]` instead of `.` is a good idea.